modules/rbac-creating-cluster-admin.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * authentication/using-rbac.adoc
+
+ifdef::openshift-enterprise,openshift-origin[]
+[id="creating-cluster-admin-{context}"]
+= Example of creating a cluster admin
```

Comment:
Is there a reason that this title starts with "Example"? Could it just be "Creating a cluster admin"?

Reply:
You're right - this flows better as starting with `Creating`. Incorporated this change.